app/templates/custom-elements/remote-screen.html

```diff
@@ -79,6 +79,20 @@
           });
         }
 
+        get mobileKeyboard() {
+          var element = this.shadowRoot.getElementById("mobile-keyboard-input");
+          if (!element) {
+            element = document.createElement("input");
+            element.id = "mobile-keyboard-input";
+            element.style.position = "fixed";
+            element.style.bottom = "-1000px";
+            this.shadowRoot
+              .querySelector(".screen-wrapper")
+              .appendChild(element);
+          }
+          element.focus();
+        }
```

Comment:
In general, I think it's clearer to define HTML elements in HTML and CSS unless there's a good reason to avoid it.

Reply:
Done